packages/hub/routes/scheduled-payments.ts

```diff
@@ -51,6 +51,29 @@ export default class ScheduledPaymentsRoute {
     ctx.type = 'application/vnd.api+json';
   }
 
+  async list(ctx: Koa.Context) {
+    if (!ensureLoggedIn(ctx)) {
+      return;
+    }
+
+    let prisma = await this.prismaManager.getClient();
+
+    let minPayAt = new Date((ctx.query['filter[payAt][gt]'] as string) || 0);
```

Comment:
Regarding `filter[payAt][gt]` query param style for filtering by range - I didn't find an existing pattern for this in the code but it seemed like a good way to do it. JSON:API is agnostic about the filter param implementation.

Reply:
maybe `pay-at` makes more sense? Since dasherised aligns with the format elsewhere.

Reply:
I don’t have strong opinions but would it be more elegant to use a timestamp for the value vs an entire ISO date string? 🤔

Reply:
Agree with pay-at, will make that change 👍

Regarding using the date string - I prefer that format over timestamps in query params because it's easier to debug and test when inspecting the URL (can quickly identify the date)